backends-velox/src/main/scala/org/apache/spark/sql/execution/datasources/velox/VeloxFormatWriterInjects.scala

```diff
@@ -47,6 +47,16 @@ trait VeloxFormatWriterInjects extends GlutenFormatWriterInjectsBase {
       nativeConf: java.util.Map[String, String]): OutputWriter = {
     val originPath = path
 
+    // Create the hdfs path if not existed.
+    val hdfsPrefix = "hdfs:"
```

Comment:
Nit:
I think it is better to use: `val hdfsScheme = "hdfs://"`

Reply:
@PHILO-HE Updated.